proxy/request_handler_test.go

```diff
@@ -491,6 +496,7 @@ func TestInitializeSession(t *testing.T) {
 				RegexpCaptureGroups: []string{"user"},
 				URL:                 x.ParseURLOrPanic("http://localhost/user"),
 				Method:              "GET",
+				Header:              newTestHeader(),
```

Comment:
i think for expected results better to use constant / or literal instead of function